dataprofiler/profilers/profiler_options.py

```diff
@@ -11,6 +11,7 @@
 from dataprofiler.profilers.json_decoder import load_option
 
 from ..labelers.base_data_labeler import BaseDataLabeler
+from ..labelers.data_labelers import DataLabeler
 
 BaseOptionT = TypeVar("BaseOptionT", bound="BaseOption")
 BooleanOptionT = TypeVar("BooleanOptionT", bound="BooleanOption")
```

Comment:
I'm curious if we needed all of these or just `BaseOptionT`

Reply:
Removing the others lead to mypy issues